### ISSUE TITLE
query results dissapears if SELECT is followed by CREATE statement

### ISSUE THREAD

**Issue by bam80** [sqlitebrowser/sqlitebrowser#2100]
Details for the issue
--------------------

#### What did you do?
In the Execute SQL tab, if SELECT is followed by CREATE statement, the query executed successfully, but no results seen.

#### What did you expect to see?
There should be query results as usual.

#### What did you see instead?
The field with results is empty.

Useful extra information
-------------------------

#### What operating system are you using?

- [ ] Windows: ( _version:_ ___ )
- [ ] Linux: ( _distro:_ ___ )
- [ ] macOS: ( _version:_ ___ )
- [ ] Other: ___

#### What is your DB4S version?

- [ ] 3.11.2
- [ ] 3.11.1
- [ ] 3.10.1
- [ ] Other: ___

#### Did you also

- [ ] Try out the latest nightly version: https://github.com/sqlitebrowser/sqlitebrowser#nightly-builds
- [ ] Search for an existing similar issue: https://github.com/sqlitebrowser/sqlitebrowser/issues?utf8=%E2%9C%93&q=is%3Aissue%20

**Comment by saikat-here** [comment]
@bam80 , just wanted to confirm, are you selecting from the same table? Is your table populated?

I have created a table and selected from a different table (which is populated) and that worked for me. 

![image](https://user-images.githubusercontent.com/2095290/73374862-ecbae400-42e0-11ea-8e73-c17fc26330da.png)

Am I following you correctly?

**Comment by bam80** [comment]
Is doesn't matter if it the same table or not. Please follow the same order statement as in the issue to reproduce it.

Example for existing `flag` table:
```sql
SELECT * FROM flag;
CREATE TEMP TABLE if not EXISTS flag(jump INTEGER);

**Comment by saikat-here** [comment]
@bam80 , correct, it's reproducible. Mostly, the display window is getting reset with the 2nd result.

**Comment by mgrojo** [comment]
This is by design. Each query resets the table with its results. A CREATE statement returns no rows. What would you expect? If we did not reset the results table with each statement, select statements returning 0 rows could be confused with previous select statement that returned rows.

**Comment by bam80** [comment]
As I understand, only SELECT statements can return rows.
As such, they and only they should always reset the results.
So I see no confusion here.
